tools/maint/gen_struct_info.py

```diff
@@ -77,6 +77,31 @@
 QUIET = (__name__ != '__main__')
 DEBUG = False
 
+CFLAGS = [
+    # Avoid parsing problems due to gcc specifc syntax.
```

Comment:
Typo in specifc (I know its in the original,  and its almost certainly mine, but still worth fixing).